{"name": "test_login_001", "status": "broken", "statusDetails": {"message": "TypeError: 'SubRequest' object is not callable", "trace": "request = <SubRequest 'browser' for <Function test_login_001>>\n\n    @pytest.fixture()\n    def browser(request):\n>       return request(\"--browser\")\nE       TypeError: 'SubRequest' object is not callable\n\ntestCases\\conftest.py:11: TypeError"}, "start": 1691723593525, "stop": 1691723593525, "uuid": "55beeb32-3c3c-42ae-b6ad-d089f169959e", "historyId": "f0eac49645079c368a84343a832d32ab", "testCaseId": "f0eac49645079c368a84343a832d32ab", "fullName": "testCases.test_login.Test_Login#test_login_001", "labels": [{"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_Login"}, {"name": "host", "value": "DESKTOP-HFVASSE"}, {"name": "thread", "value": "7480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login"}]}